sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Release History
 
-## 1.1.0-preview.1 (Unreleased)
+## 1.0.1
```

Comment:
Let's revert the version change as it's not required for dev feed release.

Reply:
version reverted. Thank you!